ts_src/transaction_builder.ts

```diff
@@ -940,6 +975,15 @@ function build(
         },
       });
     }
+    case SCRIPT_TYPES.CP2PKH: {
+      if (pubkeys.length === 0) break;
+      if (signatures.length === 0) break;
+
+      return payments.cp2pkh({
+        pubkey: pubkeys[0],
+        signature: signatures[0],
+      });
+    }
   }
```

Comment:
`SCRIPT_TYPES.CP2SH` doesn't seem to exist.

Reply:
Added at 6f8851b